spacer_floating_platform:
  FloatingPlatformDirectValveControl:
    ros__parameters:
      device: 'jetson'
      topic_name: 'valves/input'
      gpio_ids: [15, 7, 24, 26, 31, 29, 23, 21, 19]